precompile/fee_config_manager.go

```diff
@@ -200,52 +203,34 @@ func StoreFeeConfig(stateDB StateDB, feeConfig commontype.FeeConfig, blockContex
 	if blockNumber == nil {
 		return fmt.Errorf("blockNumber cannot be nil")
 	}
+	stateDB.SetState(FeeConfigManagerAddress, feeConfigLastChangedAtKey, common.BigToHash(blockNumber))
 
-	hashes, err := getFeeConfigHashes(feeConfig)
-	if err != nil {
-		return err
-	}
 	for i := minFeeConfigFieldKey; i <= numFeeConfigField; i++ {
-		stateDB.SetState(FeeConfigManagerAddress, common.Hash{byte(i)}, hashes[i])
-	}
-
-	stateDB.SetState(FeeConfigManagerAddress, lastChangedAtKey, common.BigToHash(blockNumber))
-	return nil
-}
-
-// getFeeConfigHashes takes [feeConfig] and converts them to an array of hashes, with ordered with key indexes.
-func getFeeConfigHashes(feeConfig commontype.FeeConfig) ([]common.Hash, error) {
-	res := make([]common.Hash, minFeeConfigFieldKey+numFeeConfigField)
-	for i := minFeeConfigFieldKey; i <= numFeeConfigField; i++ {
-		var hashInput common.Hash
-		var err error
+		var input *big.Int
```

Comment:
nit: I prefer this to be a hash so that we can try to convert to hash under each key case below. Currently all these fields can be converted a big number but still I feel like it's more natural to do a specific hash conversion for each of these fields.

Reply:
Good point, more future proof. Updated.